spec/automation/unit/method_validation/vm_migrate_task_complete_spec.rb

```diff
@@ -11,11 +11,21 @@
     expect(GenericMailer).to receive(:deliver).with(:automation_notification,
                                                     hash_including(:to   => user.email,
                                                                    :from => "evmadmin@example.com"
-                                                                  )
+                                                                )
                                                    )
     attrs = ["MiqServer::miq_server=#{miq_server.id}"]
     attrs << "MiqRequestTask::vm_migrate_task=#{miq_request_task.id}"
     attrs << "vm_migrate_task_id=#{miq_request_task.id}"
     MiqAeEngine.instantiate("/Infrastructure/VM/Migrate/Email/VmMigrateTask_Complete?event=vm_migrated&#{attrs.join('&')}", user)
   end
+
+  it 'sends email to nil' do
```

Comment:
Test name is misleading as you are expecting the logic to **not** send an email in this case.  Please reword.